tools/install.py

```diff
@@ -156,6 +156,14 @@ def files(action):
 
   headers(action)
 
+def ignore_inspector_headers(files, dest, action):
```

Comment:
Why not just defining this as a closure within `def headers(action):`, i.e. move to L168, and remove the `action` formal argument. Then simple do `subdir_files('deps/v8/include', 'include/node/', ignore_inspector_headers)`